action.yml

```diff
@@ -60,41 +73,56 @@ runs:
         echo "details:"
         cat details.json
         echo "building new json object.."
-        NEWJSON=$(jq '{ "${{ inputs.step_name }}": { "name": "${{ inputs.step_name }}", "status": "${{ inputs.step_status }}", "statusIcon": "'$icon'", "details": . }}' details.json)
+        NEWJSON=$(jq '{ "${{ inputs.step_name }}": { "name": "${{ inputs.step_name }}", "status": "${{ inputs.step_status }}", "statusIcon": "'$icon'", "details_preview": "${{ inputs.step_details_preview }}", "details": . }}' details.json)
         echo $NEWJSON
         echo "building new state..."
         jq ". + $NEWJSON" state.json > updated_state.json
         cat state.json
         cat updated_state.json
         mv updated_state.json state.json
         echo "## Gruntwork Pipelines Status" > $TMPFILE
+        echo "<table><tr><th>Step</th><th>Status</th><th>Output</th></tr>" >> $TMPFILE
         IFS=$'\n'
         for item in $(cat state.json | jq -c '. | values[]'); do
             echo "single item..."
             echo $item
             name=$(echo "$item" | jq -r '.name')
             statusIcon=$(echo "$item" | jq -r '.statusIcon')
```

Comment:
`statusIcon` should be snake_case

Reply:
You may have missed my comment in the PR description; the bulk of the changes are already released in a version. I pulled them in to bring `main` up to date.

Reply:
I can make these changes in a separate PR. Need to use my changes quickly.